src/osp/UserInputHandler.h

```diff
@@ -342,7 +346,7 @@ class UserInputHandler
     */
     void scroll_delta(Vector2i offset);
 
-private:
+//private:
```

Comment:
Don't comment things out like this. Either delete the line entirely, or fix the reason why you needed to unhide these variables.

Reply:
That was for debugging something, I just forgot to uncomment it.